PRAGMA foreign_keys=OFF;
BEGIN TRANSACTION;
CREATE TABLE gse(
  ID REAL,
  title TEXT,
  gse TEXT,
  status TEXT,
  submission_date TEXT,
  last_update_date TEXT,
  pubmed_id INT,
  summary TEXT,
  type TEXT,
  contributor TEXT,
  web_link TEXT,
  overall_design TEXT,
  repeats TEXT,
  repeats_sample_list TEXT,
  variable TEXT,
  variable_description TEXT,
  contact TEXT,
  supplementary_file TEXT
);
INSERT INTO gse VALUES(110771.0,'In vivo molecular signatures of severe dengue infection revealed by viscRNA-Seq','GSE116672','Public on Nov 19 2018','2018-07-05','2019-12-31',30530648,'Dengue virus infection can result in severe symptoms including shock and hemorrhage, but an understanding of the molecular correlates of disease severity is lacking. Bulk transcriptomics on blood samples are difficult to interpret because the blood is composed of different cell types that may react differently to virus infection. Dengue virus RNA can be detected in human plasma, however identifying the cells carrying dengue virus through the bloodstream in vivo has proven challenging. Here we used our recently developed viscRNA-Seq approach to profile transcriptomes of thousands of single blood peripheral mononuclear cells from 6 human subjects with dengue fever and severe dengue, as well as to characterize the cell types associated with dengue virus in the human blood. We found that although no bulk transcriptome marker for severe dengue exists, the expression of MX2 in naive B cells, of CD163 in CD14+/CD16+ monocytes and of other genes in specific cell types is highly predictive for severe dengue. We detected virus-associated cells in the blood of two severe dengue patients with high viral load and discovered the majority of these to be B cells expressing germline IgM or IgD immunoglobulin chains and naive markers but also showing signs of activation and expression of CD69, CXCR4, and other surface receptors. In bystander B cells we detected signs of strong immune activation, parallel hypersomatic evolution and, in one severe degue subject, an anomalously large clone of highly mutated, IgG1 plasmablasts that could be reactive to dengue virus. This study presents a high-resolution molecular exploration into dengue virus infection in humans and can be generalized to any RNA virus.','Expression profiling by high throughput sequencing',unistr('Fabio,,Zanini;\u0009Makeda,L,Robinson;\u0009Derek Croote;\u0009Malaya,K,Sahoo;\u0009Ana,M,Sanz;\u0009Eliana Ortiz-Lasso;\u0009Ludwig,L,Albornoz;\u0009Fernando,R,Suarez;\u0009Jose,G,Montoya;\u0009Leslie Goo;\u0009Benjamin,A,Pinsky;\u0009Stephen,R,Quake;\u0009Shirit Einav'),NULL,'Blood cells from dengue virus infected human patients were subjected to virus-inclusive single cell RNA-Seq.',NULL,NULL,NULL,NULL,unistr('Name: Fabio Zanini;\u0009Email: fabio.zanini@fastmail.fm;\u0009Laboratory: Zanini;\u0009Institute: University of New South Wales;\u0009Address: High and Botany St;\u0009City: Kensington;\u0009State: NSW;\u0009Zip/postal_code: 2033;\u0009Country: Australia'),'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE116nnn/GSE116672/suppl/GSE116672_RAW.tar');
INSERT INTO gse VALUES(121389.0,'The major risk factors for Alzheimer’s disease: Age, Sex and Genes, modulate the microglia response to Aβ plaques (CDEP)','GSE127884','Public on Apr 23 2019','2019-03-05','2019-04-29',31018141,'Microglia are involved in Alzheimer’s disease (AD) by adopting activated phenotypes. How ageing in the absence or presence of β-amyloid (Aβ) deposition in different brain areas affects this response and whether sex and AD risk genes are involved, remains however largely unknown. Here we analyzed the gene expression profiles of more than 10,000 individual microglia cells isolated from cortex and hippocampus of male and female AppNL-G-F at 4 different stages of Aβ deposition and in age-matched control mice. We demonstrate that microglia adopt two major activated states during normal aging and after exposure to amyloid plaques. One of the responses (activated response microglia, ARM) is enhanced in particular by amyloid plaques and is strongly enriched with AD risk genes. The ARM response is not homogeneous, as subgroups of microglia overexpressing MHC type II and tissue repair genes (Dkk2, Gpnmb, Spp1) are induced upon prolonged Aβ exposure. Microglia in female mice advance faster in the activation trajectories. Similar activated states were also found in a second AD model and in human brain. We demonstrate that abolishing the expression of Apoe, the major genetic risk factor for AD, impairs the establishment of ARMs, while the second microglia response type, enriched for interferon response genes, remains unaffected. Our data indicate that ARMs are the converging point of multiple AD risk factors.','Expression profiling by high throughput sequencing',unistr('Carlo,S,Frigerio;\u0009Leen Wolfs;\u0009Nicola Fattorelli;\u0009Nicola Thrupp;\u0009Iryna Voytyuk;\u0009Inga Schmidt;\u0009Renzo Mancuso;\u0009Wei-Ting Chen;\u0009Maya Woodbury;\u0009Gyan Srivastava;\u0009Thomas Möller;\u0009Eloise Hudry;\u0009Sudeshna Das;\u0009Takaomi Saido;\u0009Eric Karran;\u0009Bradley Hyman;\u0009V,H,Perry;\u0009Ma'),NULL,'RNA-Seq of cortical and hippocampal microglia in  male APP/PS1-Apoe(null), APP/PS1, C57Bl/6 and C57Bl/6-Apoe(null) mice ;  at month 18',NULL,NULL,NULL,NULL,unistr('Name: Bart de Strooper;\u0009Email: bart.destrooper@kuleuven.be;\u0009Department: VIB-KU Leuven Center for Brain & Disease Research;\u0009Institute: KULeuven;\u0009Address: Campus Gasthuisberg, Herestraat 49, bus 602;\u0009City: Leuven;\u0009Zip/postal_code: 3000;\u0009Country: Belgium'),unistr('ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE127nnn/GSE127884/suppl/GSE127884_microglia.cdep.SeuratNorm.tsv.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE127nnn/GSE127884/suppl/GSE127884_microglia.cdep.meta.csv.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE127nnn/GSE127884/suppl/GSE127884_microglia.cdep.raw.tsv.gz'));
INSERT INTO gse VALUES(121397.0,'The major risk factors for Alzheimer’s disease: Age, Sex and Genes, modulate the microglia response to Aβ plaques (KW)','GSE127892','Public on Apr 23 2019','2019-03-05','2019-08-19',31018141,'Microglia are involved in Alzheimer’s disease (AD) by adopting activated phenotypes. How ageing in the absence or presence of β-amyloid (Aβ) deposition in different brain areas affects this response and whether sex and AD risk genes are involved, remains however largely unknown. Here we analyzed the gene expression profiles of more than 10,000 individual microglia cells isolated from cortex and hippocampus of male and female AppNL-G-F at 4 different stages of Aβ deposition and in age-matched control mice. We demonstrate that microglia adopt two major activated states during normal aging and after exposure to amyloid plaques. One of the responses (activated response microglia, ARM) is enhanced in particular by amyloid plaques and is strongly enriched with AD risk genes. The ARM response is not homogeneous, as subgroups of microglia overexpressing MHC type II and tissue repair genes (Dkk2, Gpnmb, Spp1) are induced upon prolonged Aβ exposure. Microglia in female mice advance faster in the activation trajectories. Similar activated states were also found in a second AD model and in human brain. We demonstrate that abolishing the expression of Apoe, the major genetic risk factor for AD, impairs the establishment of ARMs, while the second microglia response type, enriched for interferon response genes, remains unaffected. Our data indicate that ARMs are the converging point of multiple AD risk factors.','Expression profiling by high throughput sequencing',unistr('Carlo,S,Frigerio;\u0009Leen Wolfs;\u0009Nicola Fattorelli;\u0009Nicola Thrupp;\u0009Iryna Voytyuk;\u0009Inga Schmidt;\u0009Renzo Mancuso;\u0009Wei-Ting Chen;\u0009Maya Woodbury;\u0009Gyan Srivastava;\u0009Thomas Möller;\u0009Eloise Hudry;\u0009Sudeshna Das;\u0009Takaomi Saido;\u0009Eric Karran;\u0009Bradley Hyman;\u0009V,H,Perry;\u0009Ma'),NULL,'RNA-Seq of cortical and hippocampal microglia in female and male C57Bl/6 wild type mice and in AppNL-G-F mice M3, M6, M12, M21 (32 experimental conditions, 2 mice per condition - data pooled). ',NULL,NULL,NULL,NULL,unistr('Name: Bart de Strooper;\u0009Email: bart.destrooper@kuleuven.be;\u0009Department: VIB-KU Leuven Center for Brain & Disease Research;\u0009Institute: KULeuven;\u0009Address: Campus Gasthuisberg, Herestraat 49, bus 602;\u0009City: Leuven;\u0009Zip/postal_code: 3000;\u0009Country: Belgium'),unistr('ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE127nnn/GSE127892/suppl/GSE127892_microglia.kw.SeuratNorm.tsv.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE127nnn/GSE127892/suppl/GSE127892_microglia.kw.meta.csv.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE127nnn/GSE127892/suppl/GSE127892_microglia.kw.raw.tsv.gz'));
INSERT INTO gse VALUES(121398.0,'The major risk factors for Alzheimer’s disease: Age, Sex and Genes, modulate the microglia response to Aβ plaques','GSE127893','Public on Apr 23 2019','2019-03-05','2019-08-19',31018141,'This SuperSeries is composed of the SubSeries listed below.','Expression profiling by high throughput sequencing',NULL,NULL,'Refer to individual Series',NULL,NULL,NULL,NULL,unistr('Name: Bart de Strooper;\u0009Email: bart.destrooper@kuleuven.be;\u0009Department: VIB-KU Leuven Center for Brain & Disease Research;\u0009Institute: KULeuven;\u0009Address: Campus Gasthuisberg, Herestraat 49, bus 602;\u0009City: Leuven;\u0009Zip/postal_code: 3000;\u0009Country: Belgium'),NULL);
INSERT INTO gse VALUES(130356.0,'Stem cell derived human microglia transplanted in mouse brain to study human disease','GSE137444','Public on Oct 16 2019','2019-09-13','2020-01-15',31659342,'While genetics highlight the role of microglia in Alzheimer’s disease (AD), one third of putative AD-risk genes lack adequate mouse orthologs. Here, we successfully engraft human microglia derived from embryonic stem cells in the mouse brain. The cells recapitulate transcriptionally human primary microglia ex vivo and show expression of human specific AD-risk genes. Oligomeric Amyloid-β induces a divergent response in human vs. mouse microglia. This model can be used to study the role of microglia in neurological diseases.','Expression profiling by high throughput sequencing',unistr('Renzo,,Mancuso;\u0009Johanna Van Den Daele;\u0009Nicola Fattorelli;\u0009Leen Wolfs;\u0009Sriram Balusu;\u0009Oliver Burton;\u0009Annerieke Sierksma;\u0009Yannick Fourne;\u0009Suresh Poovathingal;\u0009Amaia Arranz-Mendiguren;\u0009Carlo Sala Frigerio;\u0009Christel Claes;\u0009Lutgarde Serneels;\u0009Tom Theys;\u0009Hugh P'),NULL,'To control for variability across different experiments, tissue from three mice was pooled in all experimental conditions, from different litters and transplanted with H9-microglia produced in independent batches of in vitro differentiation. Across the manuscript we report both the number of mice/differentiations pooled, as well as the number of sequencing pools. In summary, 1) we sequenced 2,246 transplanted H9-microglia (n=3/1, 3 mice in 1 combined sequencing pool), 4496 H9-derived monocytes (n=2/1, 2 differentiations in 1 combined sequencing pool) and 3385 microglia in vitro (n=2/1), and 22,846 human primary microglia obtained from cortical surgical resections. 2) We sequenced 4880 transplanted H9-microglia (n=3x2/2) and 9942 host mouse cells (n=3x2/2) and assessed their differential response to oligomeric amyloid beta.',NULL,NULL,NULL,NULL,unistr('Name: Mark Fiers;\u0009Email: mark.fiers@kuleuven.vib.be;\u0009Department: Center for Brain and Disease;\u0009Institute: VIB;\u0009Address: Herestraat 49;\u0009City: Leuven;\u0009Zip/postal_code: 3000;\u0009Country: Belgium'),unistr('ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE137nnn/GSE137444/suppl/GSE137444_chimera_human_h9.tsv.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE137nnn/GSE137444/suppl/GSE137444_chimera_mouse.tsv.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE137nnn/GSE137444/suppl/GSE137444_human_patient.tsv.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE137nnn/GSE137444/suppl/GSE137444_invitro_mh_mc.tsv.gz'));
INSERT INTO gse VALUES(138022.0,'Single cell transcriptome from aplastic anemia (AA) and healthy donors [Full-length RNA-seq]','GSE145531','Public on Mar 31 2021','2020-02-19','2021-03-31',33763704,'Transcriptome analysis of hematopoietic stem and progenitor cells (HSPCs) from aplastic anemia (AA) and healthy donors','Expression profiling by high throughput sequencing',unistr('Caiying,,Zhu;\u0009Chenchen Wang;\u0009Yu Lian;\u0009Peng Wu;\u0009Tao Cheng;\u0009Jun Shi;\u0009Ping Zhu'),NULL,'Full length mRNA profiles of hematopoietic stem and progenitor cells (HSPCs) under aplastic anemia (AA) and healthy conditions were generated by deep sequencing at single cell resolution, using Illumina Hiseq',NULL,NULL,NULL,NULL,unistr('Name: Caiying Zhu;\u0009Email: zhucaiying@ihcams.ac.cn;\u0009Phone: 13622053408;\u0009Institute: Institute of Hematology & Blood Diseases Hospital;\u0009Address: NO. 288 Nanjing Road, Heping District, Tianjin;\u0009City: Tianjin;\u0009Zip/postal_code: 300020;\u0009Country: China'),'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE145nnn/GSE145531/suppl/GSE145531_Full_length_SingleCell_counts.txt.gz');
INSERT INTO gse VALUES(138150.0,'Single cell transcriptome from aplastic anemia (AA) and healthy donors [3'' RNA-seq]','GSE145668','Public on Mar 31 2021','2020-02-21','2021-03-31',33763704,'Transcriptome analysis of hematopoietic stem and progenitor cells (HSPCs) and T cells collected from bone marrow and peripheral blood of healthy donors and aplastic anemia patients untreated or response to immunosuppressive therapy.','Expression profiling by high throughput sequencing',unistr('Caiying,,Zhu;\u0009Chenchen Wang;\u0009Yu Lian;\u0009Peng Wu;\u0009Xuan Li;\u0009Jun Shi;\u0009Ping Zhu;\u0009Tao Cheng'),NULL,'3'' end mRNA profiles of hematopoietic stem and progenitor cells (HSPCs) and T cells under aplastic anemia (AA) and healthy conditions were generated by deep sequencing at single cell resolution, using Illumina Hiseq',NULL,NULL,NULL,NULL,unistr('Name: Caiying Zhu;\u0009Email: zhucaiying@ihcams.ac.cn;\u0009Phone: 13622053408;\u0009Institute: Institute of Hematology & Blood Diseases Hospital;\u0009Address: NO. 288 Nanjing Road, Heping District, Tianjin;\u0009City: Tianjin;\u0009Zip/postal_code: 300020;\u0009Country: China'),unistr('ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE145nnn/GSE145668/suppl/GSE145668_AA_cells_counts.txt.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE145nnn/GSE145668/suppl/GSE145668_Ctrl_cells_counts.txt.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE145nnn/GSE145668/suppl/GSE145668_Response_cells_counts.txt.gz'));
INSERT INTO gse VALUES(138151.0,'Transcriptome analysis of HSPCs and T cells from aplastic anemia (AA) and healthy donors','GSE145669','Public on Mar 31 2021','2020-02-21','2021-03-31',33763704,'This SuperSeries is composed of the SubSeries listed below.','Expression profiling by high throughput sequencing',NULL,NULL,'Refer to individual Series',NULL,NULL,NULL,NULL,unistr('Name: Caiying Zhu;\u0009Email: zhucaiying@ihcams.ac.cn;\u0009Phone: 13622053408;\u0009Institute: Institute of Hematology & Blood Diseases Hospital;\u0009Address: NO. 288 Nanjing Road, Heping District, Tianjin;\u0009City: Tianjin;\u0009Zip/postal_code: 300020;\u0009Country: China'),NULL);
INSERT INTO gse VALUES(138483.0,'single cell RNA-seq analysis of ovarian cancer ascites','GSE146026','Public on May 01 2020','2020-02-27','2020-06-26',32572264,'To understand the cellular diversity within malignant ascites, we profiled ~11,000 single cells from 22 samples by Smartseq2 and by 10x.','Expression profiling by high throughput sequencing','Itay,,Tirosh',NULL,unistr('Ascites samples were either depleted of CD45+ cells and profiled by 10x or sorted by EPCAM+CD24+ and profiled by SmartSeq2;\u0009Note from Submitter: We have difficulties in approving raw data submission of data from clinical patient samples to GEO or dbGAP and are therefore submitting at this point only the processed data and the metadata.  In parallel with the process of granting approval for raw data submission to dbGAP we are also using another repository to which we are allowed to deposit raw data (DUOS@broadinstitute.org).'),NULL,NULL,NULL,NULL,unistr('Name: Itay Tirosh;\u0009Email: Tirosh.itay@gmail.com;\u0009Institute: WEIZMANN INSTITUTE OF SCIENCE;\u0009Address: Herzl 234;\u0009City: Rehovot;\u0009State: NA;\u0009Zip/postal_code: 7610001;\u0009Country: Israel'),unistr('ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE146nnn/GSE146026/suppl/GSE146026_Izar_HGSOC_ascites_10x_log.tsv.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE146nnn/GSE146026/suppl/GSE146026_Izar_HGSOC_ascites_SS2_log.tsv.gz'));
INSERT INTO gse VALUES(156272.0,'Bulk RNA-seq of aplastic anemia (AA) and healthy donors','GSE165870','Public on Mar 31 2021','2021-01-31','2021-03-31',33763704,'Bulk RNA-seq data of  Lin-CD34+ hematopoietic stem and progenitor cells derived from bone marrow of healthy donors and untreated aplastic anemia  patients','Expression profiling by high throughput sequencing',unistr('Caiying,,Zhu;\u0009Chenchen Wang;\u0009Yu Lian;\u0009Peng Wu;\u0009Tao Cheng;\u0009Jun Shi;\u0009Ping Zhu'),NULL,'Mononuclear cells from 3 healthy donors and untreated aplastic anemia patients'' bone marrow aspirates were isolated using Ficoll density gradient separation and cryopreserved in 90% FBS/ 10% DMSO for storage in liquid nitrogen. The thawing frozen cells were stained with human antibodies including lineage cocktail (CD3, CD14, CD16, CD19, CD20, CD56) (BV510, Biolegend) and CD34 (APC, Clone 581, BD Biosciences) antibodies. Lin-CD34+ HSPCs (cell count ranges from 130 to 2000) were sorted directly into cell lysis buffer on a BD FACSAria III. The reverse transcription and template switch steps were performed following the full-length Smart-seq2 protocol with some modifications. Concentration of oligd(T30VN) and template switch oligo (TSO) primers used for mRNA reverse transcription were double. The cDNA was fragmented using Covaris S220 instead of Tn5 transposase. The processes of end repair, polyA-tailing, adaptor ligation and library amplification were using KAPA Hyper Prep Kits (Kapa Biosystems, Cat# KK8504) and NEBNext Multiplex Oligos for Illumina (Index Primers Set 1) (NEB, Cat# E7335L).The 150bp paired-end reads were generated from Illumina NovaSeq6000 platform. For data processing, TSO primer, polyA tail, adaptor, and low-quality sequence (N > 10%) were trimmed using Trimmomatic (version 0.36). Clean reads were aligned against the human GRCh38 genome using STAR (version 2.7)). Duplicate reads were identified and removed by using Picard(version 2.17.6). MATS(version 4.0.2) and DaPars(version 0.9.0) to perform alternative splicing and alternative polyadenylation analysis, respectively. Gene quantification was using HTSeq (version 0.9.1).',NULL,NULL,NULL,NULL,unistr('Name: Caiying Zhu;\u0009Email: zhucaiying@ihcams.ac.cn;\u0009Phone: 13622053408;\u0009Institute: Institute of Hematology & Blood Diseases Hospital;\u0009Address: NO. 288 Nanjing Road, Heping District, Tianjin;\u0009City: Tianjin;\u0009Zip/postal_code: 300020;\u0009Country: China'),'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE165nnn/GSE165870/suppl/GSE165870_Full_length_bulk_counts.txt.gz');
INSERT INTO gse VALUES(219747.0,'Xenografted human microglia display diverse transcriptomic states in response to Alzheimer’s disease-related Aβ pathology','GSE216999','Public on Feb 29 2024','2022-11-01','2024-02-29',NULL,'Microglia are central players in Alzheimer’s Disease (AD) pathology, but analyzing microglia states in human brain samples is challenging due to genetic diversity, postmortem delay and admixture of pathologies. To circumvent these issues, here we generated 138,577 single cell expression profiles of human stem cell-derived microglia xenotransplanted in the brain of the AppNL-G-F model of amyloid pathology and wild type controls. Xenografted human microglia adopt a disease-associated (DAM) profile similar to that seen in mouse microglia, but display a more pronounced HLA state, likely related to antigen presentation in response to amyloid plaques. The human microglial response also involves a pro-inflammatory cytokine/chemokine CRM response to oligomeric Aβo. Genetic deletion of TREM2 or APOE, as well as APOE polymorphisms and TREM2R47H expression in the transplanted microglia modulate these responses differentially. The expression of other AD risk genes is differentially regulated across the distinct cell states elicited in response to amyloid pathology. Thus, we have identified multiple transcriptomic cell states adopted by human microglia in response to AD-related pathology, which should be taken into account in translational studies.','Expression profiling by high throughput sequencing',unistr('Renzo,,Mancuso;\u0009Nicola Fattorelli;\u0009Anna Martinez-Muriana;\u0009Emma Davis;\u0009Leen Wolfs;\u0009Johanna Van Den Daele;\u0009Ivana Geric;\u0009Jessie Premereur;\u0009Paula Polanco;\u0009Baukje Bijnens;\u0009Pranav Preman;\u0009Lutgarde Serneels;\u0009Suresh Poovanthigal;\u0009Sriram Balusu;\u0009Catherine Verfaill'),'https://mancusolab.bioinf.be/Mancuso2024/','The main goal of this project is to decipher the complex human microglial response towards amyloid aggregates. Here, we transplanted human microglia progenitors from several stem cell lines (UKBIO11-A, BIONi010-C and H9) into the mouse brain of AppNL-G-F, Apphu/hu or AppNL-G-F ApoE-/-  mice to mimic several amyloid pathology conditions, including injection of oligomeric Aβo. After exposure to diseased conditions, we isolated human microglia and sequenced more than 138,000 microglia across 106 mice.',NULL,NULL,NULL,NULL,unistr('Name: Bart De Strooper;\u0009Email: bart.destrooper@kuleuven.be;\u0009Laboratory: Laboratory for the Research of Neurodegenerative Diseases;\u0009Department: Center for Brain & Disease Research;\u0009Institute: VIB-KU Leuven;\u0009Address: Herestraat 49 box 602;\u0009City: Leuven;\u0009Zip/postal_code: 3000;\u0009Country: Belgium'),unistr('ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE216nnn/GSE216999/suppl/GSE216999_Mancuso2022_NormExpr.mtx.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE216nnn/GSE216999/suppl/GSE216999_Mancuso2022_NormExpr_cellbarcodes.txt.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE216nnn/GSE216999/suppl/GSE216999_Mancuso2022_NormExpr_genes.txt.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE216nnn/GSE216999/suppl/GSE216999_Mancuso2022_RNAcounts.mtx.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE216nnn/GSE216999/suppl/GSE216999_Mancuso2022_RNAcounts_cellbarcodes.txt.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE216nnn/GSE216999/suppl/GSE216999_Mancuso2022_RNAcounts_genes.txt.gz;\u0009ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE216nnn/GSE216999/suppl/GSE216999_Mancuso2022_metadata.csv.gz'));
CREATE TABLE gsm(
  ID REAL,
  title TEXT,
  gsm TEXT,
  series_id TEXT,
  gpl TEXT,
  status TEXT,
  submission_date TEXT,
  last_update_date TEXT,
  type TEXT,
  source_name_ch1 TEXT,
  organism_ch1 TEXT,
  characteristics_ch1 TEXT,
  molecule_ch1 TEXT,
  label_ch1 TEXT,
  treatment_protocol_ch1 TEXT,
  extract_protocol_ch1 TEXT,
  label_protocol_ch1 TEXT,
  source_name_ch2 TEXT,
  organism_ch2 TEXT,
  characteristics_ch2 TEXT,
  molecule_ch2 TEXT,
  label_ch2 TEXT,
  treatment_protocol_ch2 TEXT,
  extract_protocol_ch2 TEXT,
  label_protocol_ch2 TEXT,
  hyb_protocol TEXT,
  description TEXT,
  data_processing TEXT,
  contact TEXT,
  supplementary_file TEXT,
  data_row_count REAL,
  channel_count REAL
);
INSERT INTO gsm VALUES(2894209.0,'1001701101_A1','GSM3245112','GSE116672','GPL18573','Public on Nov 19 2018','2018-07-05','2018-11-19','SRA','blood (PBMCs)','Homo sapiens',unistr('tissue: blood;\u0009patient id: 3-013;\u0009cell type: T cell;\u0009Sex: M;\u0009diseaseseverity: control;\u0009numberhumanuniquelymappedreads: 104882;\u0009numbervirusreads: 0;\u0009age: 14'),'total RNA',NULL,NULL,unistr('No extraction, single cells are sorted into 0.4ul lysis buffer;\u0009Cells are lysed in lysis buffer containing Triton X-100, then RT using template switching oligo and preamplification for 21 cycles as in SmartSeq2 is performed. Cells with different levels of intracellular virus are cherry picked and sequencing libraries are made using illumina’s Nextera XT kit.'),NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,unistr('bcl2fastq demux;\u0009STAR mapping;\u0009Htseq-count 0.7+ gene counting;\u0009Stampy virus mapping;\u0009Custom script for filtering virus reads;\u0009Genome_build: grch38;\u0009Supplementary_files_format_and_content: One TSV for each sample containing the counts of gene expression.'),unistr('Name: Fabio Zanini;\u0009Email: fabio.zanini@fastmail.fm;\u0009Laboratory: Zanini;\u0009Institute: University of New South Wales;\u0009Address: High and Botany St;\u0009City: Kensington;\u0009State: NSW;\u0009Zip/postal_code: 2033;\u0009Country: Australia'),NULL,0.0,1.0);
INSERT INTO gsm VALUES(3709608.0,'Human Sample 1','GSM4107622','GSE137444','GPL20301','Public on Oct 16 2019','2019-10-03','2019-10-16','SRA','Temporal cortex','Homo sapiens',unistr('cell type: Microglia;\u0009human age: 45 years;\u0009human gender: Male'),'polyA RNA',NULL,NULL,unistr('Flow Cytometry or MACS.;\u000910X Genomics V2 chemistry.'),NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,unistr('RM65;\u0009processed data file: human_patient.tsv.gz'),unistr('Cellranger (version 2.1.1).;\u0009Genome_build: Human hg38 (GRCh38), Ensembl version 84; mouse mm10 (GRCm38), Ensembl version 84.;\u0009Supplementary_files_format_and_content: *.tsv.gz: Raw gene expression counts per cell in a gzipped tab-separated values file.'),unistr('Name: Mark Fiers;\u0009Email: mark.fiers@kuleuven.vib.be;\u0009Department: Center for Brain and Disease;\u0009Institute: VIB;\u0009Address: Herestraat 49;\u0009City: Leuven;\u0009Zip/postal_code: 3000;\u0009Country: Belgium'),NULL,0.0,1.0);
INSERT INTO gsm VALUES(3935955.0,'10x_1','GSM4352062','GSE146026','GPL18573','Public on May 01 2020','2020-02-27','2020-05-01','SRA','Malignant ascites  single cell','Homo sapiens',unistr('cell type: Malignant ascites  single cell;\u0009patient id: 5;\u0009time point: 1'),'polyA RNA',NULL,'Ascites fluid was drained by an interventional radiologist and transferred for further processing in closed vacuum bottles. De-identified patient information, including their ovarian cancer histology, stage, treatment history, and BRCA mutation status were collected. Immediately following drainage, malignant ascites was transported on ice, aliquoted into 50 mL conical tubes (BD Falcon) and spun for 5 min at 580 × g at 4°C. The supernatant was aspirated and the remaining pellet was resuspended with 5 mL of hypotonic lysis buffer ACK (Life technologies) and incubated on ice for 5 min. 20 ml of PBS was used to quench the lysis buffer. Cell suspensions were pooled and pipetted into a new 50 mL conical tube through a 100 µm mesh and spun for 5 min at 580 × g at 4°C. Hypotonic lysis was repeated until no visible red blood cell component was present, usually for a total of 2-3 times (for the plate-based approach). Following red blood cell lysis, the cell pellet was resuspended in PBS with 2% (v/v) FBS. These cells were then used for both plate-based and droplet-based scRNA-seq. For plate-based scRNA-seq, cells were labeled with the following fluorophore-conjugated flow-cytometry antibodies: live/dead stain with Calcein-AM (LifeTechnologies), 7-AAD (7-Aminoactinomycin D) (LifeTechnologies), CD45-FITC (LifeTechnologies), EpCAM-PE (Miltenyi Biotec) and CD24-PE/Cy7 (BioLegend). Cells were incubated for 30 min on ice in the dark. Cells were washed twice by resuspending with PBS with 2% (v/v) FBS and spun 5 min at 580 × g at 4°C. Flow-cytometry and sorting was performed on a BD Biosciences cell sorter (IIu or UV). Following doublet exclusion, Calceinhigh7AAD-CD45-EPCAM+CD24+ single cells were sorted into 96-well microtiter plates (on a plate chiller) that were prepared with 10uL cell lysis buffer (TCL + 1% β-mercaptoethanol). Following completion of cell sorting, plates were covered with aluminum lids, vortexed for 10 seconds, centrifuged for 2 min at 580 × g at 4°C and immediately placed on dry ice prior to storage at -80°C.',unistr('RNA from single cells was first purified with Agencourt RNAClean XP beads (Beckman Coulter);\u0009Libraries from isolated single cells were generated based on the Smart-seq2 protocol (Picelli  2014) with the following modifications.oligo-dT were used to prime reverse transcription with Maxima reverse transcriptase and locked TSO oligonucleotide, which was followed by 20 cycle PCR amplification using KAPA HiFi HotStart ReadyMix (KAPA Biosystems) with subsequent Agencourt AMPure XP bead purification as described. Libraries were tagmented using the Nextera XT Library Prep kit (Illumina) with custom barcode adapters (sequences available upon request). Libraries from 384 cells with unique barcodes were combined and sequenced using a NextSeq 500 sequencer (Illumina). For a subset of samples, single cells were processed through the 10X Chromium 3 Single Cell Platform using the Chromium Single Cell 3’ Library, Gel Bead and Chip Kits (10X Genomics, Pleasanton, CA), following the manufacturer’s protocol. Briefly, cells were added to each channel of a chip to be partitioned into Gel Beads in Emulsion (GEMs) in the Chromium instrument, followed by cell lysis and barcoded reverse transcription of RNA in the droplets. Breaking of the emulsion was followed by amplification, fragmentation and addition of adapter and sample index.'),NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,'Izar_HGSOC_ascites_10x_log.tsv',unistr('Paired-end, 38-base reads were mapped to the UCSC hg19 human transcriptome using Bowtie with parameters "-q --phred33-quals -n 1 -e 99999999 -l 25 -I 1 -X 2000 -a -m 15 -S -p 6". Expression values were calculated by   RSEM v1.2.3 in paired-end mode using parameters "--estimate-rspd --paired end -sam -p 6", from which TPM values for each gene  were extracted. For the subset of cells processed with 10X, we used CellRanger with the default parameters.;\u0009We examined as quality control the number of genes detected in each cell and conservatively excluded cells with fewer than 2,000 detected genes.;\u0009Genome_build: hg19;\u0009Supplementary_files_format_and_content: Two tab-delimited files containing log2(TPM/10+1) values (TPM=transcript-per-million reads) of 9609 cells profiled by 10x droplet scRNA-seq (10x file) and of 1297 cell profiled by SmartSeq2 (SS2 file). All quantified 23,686 genes are included in the SS2 file and 11,548 genes detected in at least 10 cells are included in the 10x file. Header lines include Cell ID, patient ID, time point of sample acquisition, and the cells assignment into clusters and coordinates in a tSNE analysis (as shown in an associated manuscript).'),unistr('Name: Itay Tirosh;\u0009Email: Tirosh.itay@gmail.com;\u0009Institute: WEIZMANN INSTITUTE OF SCIENCE;\u0009Address: Herzl 234;\u0009City: Rehovot;\u0009State: NA;\u0009Zip/postal_code: 7610001;\u0009Country: Israel'),NULL,0.0,1.0);
INSERT INTO gsm VALUES(4262274.0,'P2Res_B3_sc01','GSM4701141','GSE145668','GPL20301','Public on Mar 31 2021','2020-07-29','2021-03-31','SRA','P2Res','Homo sapiens',unistr('Sex: Female;\u0009treatment: treated;\u0009condition: non-SAA;\u0009surface markers: Lin-CD34+;\u0009tissue: Bone Marrow;\u0009age: 25'),'total RNA',NULL,NULL,unistr('Fresh bone marrow and peripheral blood mononuclear cells were isolated by the Ficoll-PaqueTM PLUS (GE Healthcare) following the manufacturer’s instruction. Bone marrow samples were divided into two parts. One part was used for sorting HSPCs which were stained with human antibodies including lineage cocktail (CD3, CD14, CD16, CD19, CD20, CD56) (BV510, Biolegend), CD34 (APC, Clone 581, BD Biosciences), CD38 (PE-Cy™7, clone HIT2, BD Biosciences), CD90 (PerCP-Cy™5.5, clone5E10, BD Biosciences), CD45RA (FITC, clone HI100, BD Biosciences), CD49f (BV605, clone GoH3, BD Bioscience), CD10 (BV786, clone HI10a, BD Biosciences), CD135 (PE, clone 4G8, BD Biosciences). Another part of bone marrow and all peripheral blood were used for sorting T cells which were stained antibodies including CD3 (APC-Cy™7, clone SK7, BD Biosciences), CD4 (APC, clone RPA-T4, BD Biosciences), CD8 (PE-Cy™7, clone RPA-T8, BD Biosciences). All samples were incubated on ice for 30mins in the dark and washed with PBS. After cell suspension, DAPI (1 ug/ml, Sigma-Aldrich) was added to remove dead cells for FACS. Flow cytometry sorting was performed on a BD FACSAria III (BD Bioscience).;\u0009Fresh isolated cells were sorted directly into cell lysis buffer on a BD FACSAria III. The reverse transcription and template switch steps were performed following the modified Smart-seq2 protocol.(STRT-seq). Single cells with different barcodes were pooled for one library. The mixed cDNA was purified using DNA Clean and Concentration Kit (ZYMO, Cat.#D4014), and the small fragments and primers were removed by 0.8X AMPure XP beads (Beckman, Catt#A63882). A total of 30~40 ng purified products were amplified using biotinylated primers anchored with 8 nt sequences for library indexing. 300~450 ng cDNA was fragmented into 300 bp using Covaris S220 (Covaris). Dynabeads MyOne Streptavidin C1 (Invitrogen, Cat#65001) was used to enrich the 3'' end of fragmented cDNA. The processes of end repair, polyA-tailing, adaptor ligation and library amplification were using KAPA Hyper Prep Kits (Kapa Biosystems, Cat#KK8504) and NEBNext Multiplex Oligos for Illumina (Index Primers Set 1) (NEB, Cat#E7335L).;\u0009Modified Smart-seq2'),NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,'Response_cells_counts.txt',unistr('Sequencing data was demultiplexed based on barcodes in Read 2 (R2) of paired-end reads. UMI in R2 was added to the head of corresponding read 1 (R1), which was then trimmed to remove the template switch oligo (TSO) and ployA tail sequences; we also trimmed the reads with adaptor contaminants and low-quality bases (N> 10%).;\u0009The clean reads were aligned to the human GRCh38 genome using hisat2 (v2.1.0) with the known gene annotation (GENCODE release 27). Gene quantification was based on the UMIs after removing duplicated reads using modified HTSeq (version 0.9.1);\u0009Gene expression level was normalized as log2 (TPM/10 + 1), in which TPM is short for Transcripts per million and adding one is to avoid negative expression values.;\u0009Genome_build: GRCh38;\u0009Supplementary_files_format_and_content: tab-delimited text files include count values for each single cells.'),unistr('Name: Caiying Zhu;\u0009Email: zhucaiying@ihcams.ac.cn;\u0009Phone: 13622053408;\u0009Institute: Institute of Hematology & Blood Diseases Hospital;\u0009Address: NO. 288 Nanjing Road, Heping District, Tianjin;\u0009City: Tianjin;\u0009Zip/postal_code: 300020;\u0009Country: China'),NULL,0.0,1.0);
CREATE TABLE gse_gsm(gse TEXT,gsm TEXT);
INSERT INTO gse_gsm VALUES('GSE116672','GSM3245112');
INSERT INTO gse_gsm VALUES('GSE137444','GSM4107622');
INSERT INTO gse_gsm VALUES('GSE146026','GSM4352062');
INSERT INTO gse_gsm VALUES('GSE145668','GSM4701141');
COMMIT;
